Dockerfile

```
@@ -4,7 +4,7 @@ LABEL org.opencontainers.image.source=https://github.com/SAP/devops-docker-neo-c
 LABEL org.opencontainers.image.description="An image for the SAP neo cli"
 LABEL org.opencontainers.image.licenses=Apache-2.0
 
-ARG NEO_SDK_VERSION=1.185.3
+ARG NEO_SDK_VERSION=1.120.2
```

```suggestion
ARG NEO_SDK_VERSION=1.196.6
```
Did a downgrade there :/